Add tests (#9)
BullSequana/portails4: examples/README.md

## Examples

In this folder, you can find examples where the Portals functions are used.

#

* **init_queue** : show the initalization of the interface, setup a buffer for incoming messages and get an event into the event queue.

* **put_to_self** : show the sending of a message to oneself and handle completion events.

> * **hello** : show the connection between a transmiter and a receiver.
>     - client : send a message and receive it's ack.
>     - server : receive the message and read it.

Reviewer: ```suggestion
* **hello** : send an hello world from a client to a server (physical addressing).
    - client : send a message and receive it's ack.
    - server : receive the message and read it.
```
